src/common/components/DataTable/FilterDropdown.tsx

```diff
@@ -166,6 +194,16 @@ export const FilterDropdown: FC<{
     else setOpenFilter(filter);
   };
 
+  const checkIfActiveFilter = (filter: FilterInfo): string => {
```

Comment:
The naming of this function is a bit weird. One would think this returns a boolean.

I'd probably prefer `isFilterActive` which returns  `bool` and then do the logic of what class to put in the template.